### PR TITLE
feat: enhance styling and ESLint compliance

### PR DESCRIPTION
**Pull request by stone-lyl**
- feat: ensure that the scrollbar remains light in dark mode
- feat: enhance parameter configuration pill interface

| Before | After |
| ------ | ----- |
| ❌      | ✅    |
|image|image|
|image |image|

- feat: update ESLint configuration

image

### CODE REVIEW

File: packages/ds-ext/src/DiagramEditorProvider.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export class DiagramEditorProvider implements vscode.CustomEditorProvider<Diagra
       };
 
       const handler = handlers[event.type];
-      if(!handler) throw Error(`No handler found for event type: ${event.type}. Available handlers: ${Object.keys(handlers).join(', ')}`);
```
Comment (ajthinking) [inline packages/ds-ext/src/DiagramEditorProvider.ts:62]: I think we should not enforce brackets (setting: curly 'warn'?) as I think no-brackets makes sense in some situations, like early exits for instance.

File: packages/ds-ext/src/app/VsCodeClient.tsx [hunk 2]
```diff
@@ -66,22 +66,22 @@ export class VsCodeClient implements WorkspaceApiClient {
   private handleMessage(data: any) {
     processWaitingResponse(data);
 
-    if (data.awaited) return;
+    if (data.awaited) {return;}
 
     // ...If message is non-transactional, handle it
     if (data.type === 'ExecutionUpdate') {
-      this.updateEdgeCounts!(data.counts)
+      this.updateEdgeCounts!(data.counts);
```
Comment (ajthinking) [inline packages/ds-ext/src/app/VsCodeClient.tsx:73]: Prefer no brackets

**Review by ajthinking — APPROVED** [review]
Nice work👌

**Comment by stone-lyl** [comment]
> handlers[event.type];

image

File: packages/ds-ext/src/DiagramEditorProvider.ts (hunk 1 above)
Comment (stone-lyl) [reply packages/ds-ext/src/DiagramEditorProvider.ts]: I have updated the two suggestions in my code.

**Review by ajthinking — APPROVED** [review]

**Pull request merged**